src/functions/operatorname.js

```diff
@@ -42,47 +38,78 @@ defineFunction({
                 }
             });
 
-            // Consolidate Greek letter function names into symbol characters.
-            const temp = html.buildExpression(
+            // Consolidate function names into symbol characters.
+            const expression = html.buildExpression(
                 groupValue, options.withFont("mathrm"), true);
 
-            // All we want from temp are the letters. With them, we'll
-            // create a text operator similar to \tan or \cos.
-            for (const child of temp) {
+            for (const child of expression) {
                 if (child instanceof domTree.symbolNode) {
-                    letter = child.value;
-
-                    // In the amsopn package, \newmcodes@ changes four
-                    // characters, *-/:’, from math operators back into text.
-                    // Given what is in temp, we have to address two of them.
-                    letter = letter.replace(/\u2212/, "-");   // minus => hyphen
-                    letter = letter.replace(/\u2217/, "*");
-
-                    // Use math mode for Greek letters
-                    mode = (/[\u0391-\u03D7]/.test(letter) ? "math" : "text");
-                    output.push(buildCommon.mathsym(letter, mode));
-                } else {
-                    output.push(child);
+                    let letter = child.value;
+                    if (/[\u2212\u2217]/.test(letter)) {
```

Comment:
Is this check necessary?

Reply:
That check is still necessary but the similar one above has no effect. So I removed the ineffective code.

Reply:
@ronkok Doesn't `replace` do nothing if those letters don't exist?

Reply:
> Doesn't `replace` do nothing if those letters don't exist?

I don't think that's the relevant question. Those letters *do* exist.  Say that an author inputs `\operatorname{atan*}`.  During `html.buildExpression()` above, that asterisk, *, U+002A, is converted to Asterisk Operator, ∗, U+2217. Since we're emulating the amsopn package and it reverts that operator back to a text asterisk, we should do the same thing.

Reply:
@ronkok I meant, there won't be much difference, but can we just do:

```js
child.value = child.value.replace(/\u2212/, "-")
    .replace(/\u2217/, "*");
```

without `/[\u2212\u2217]/.test(letter)` check?

Reply:
Sure. Will do.

Reply:
@ronkok Could you change this one, too?

Reply:
Will do.